### ISSUE TITLE
Pull request comments

### ISSUE THREAD

**Issue by 304NotModified** [codecov/support#200]
The pull request comments works sometimes, and sometimes not. 

How do can we fix this?

**Comment by stevepeak** [comment]
@304NotModified what an `owner/repo@sha` I can review this behavior at. Thanks!

**Comment by 304NotModified** [comment]
Nlog/nlog@c3c91c4c45e1dc9b714494f20b111d4a19179e92

See https://github.com/NLog/NLog/pull/1439

I guess https://github.com/codecov/support/issues/199 is related

**Comment by rad3ks** [comment]
Any update on that? After your migration we don't get PR comments and constantly see spinner when I check certain branch. When I click on `your bot` it open account `Yaml` settings page and there are no instructions on how to check if codecov is authorized (it was).

<img width="691" alt="screenshot 2016-05-19 00 00 46" src="https://cloud.githubusercontent.com/assets/3084262/15376360/cfb3164e-1d54-11e6-91a4-3b9f65b60611.png">

**Comment by stevepeak** [comment]
@rad3ks same project NLog?

**Comment by rad3ks** [comment]
No, it's different project, under NDA unfortunately. How can I give more information to investigate the issue? Maybe I can do it on my own? If so, how can I verify if codecov is authorized to access my github private repo (organization repo)?

**Comment by samuelcolvin** [comment]
This is still an issue and is getting very frustrating. 

Very simple PR: no complexity with merge commits, no message about bot authorisation. 

It's a private project:
* pull: https://codecov.io/gh/tutorcruncher/TutorCruncher2/pull/3326
* commit: https://codecov.io/gh/tutorcruncher/TutorCruncher2/commit/dd71120a32ff8f772443bcc8d02eb5cb2bf30ec7

The commit build log shows no errors, but doesn't mention making a comment at all.

**Comment by stevepeak** [comment]
@samuelcolvin sorry for this wildness. Looks like for this particular pull there is no coverage data uploaded for the PR base at 8444d2a8efdad1.

I'm reviewing why the comment did not post in this case, but it would have been essentially blank.

**Comment by samuelcolvin** [comment]
@stevepeak still getting a lot of problems with this, either the comment is missing or based on wrong commit.

I think this might relate to PRs where we've rebased with master?

**Comment by 304NotModified** [comment]
We also experience unstable behaviour in multiple repositories. It's less then after the upgrade, but still not as stable as before.

**Comment by eddiemoore** [comment]
I've been finding that the pull request comments are not updating. They are always referencing an old commit. And sometimes when it comments on the pull request it's referencing the commit from 2 commits ago.

**Comment by samuelcolvin** [comment]
I'm pretty sure this relates to rebasing or amending a commit, anything requiring a force push seems to break the comments. This is odd as the differ linked to in the status update seems right.

**Comment by 304NotModified** [comment]
@stevepeak bump

**Comment by stevepeak** [comment]
Hey friends, how are we looking today? We had a release that was updating pull request timestamps improperly causing then to not be updated. This did not occur on all projects, but many.

**Comment by 304NotModified** [comment]
OK great things got fixed! Will check if this issue is still active :)

**Comment by stevepeak** [comment]
Thanks! Sorry about all the craziness. I really really appreciate your patience and help 👍

**Comment by gdamore** [comment]
I'm seeing similar behavior, for example:

https://github.com/nanomsg/nng/pull/31

I have not been able to get a codecov report to show up in PR comments on this repo yet.  I can see that codecov did process the PR though, as seen here:

https://codecov.io/gh/nanomsg/nng/pull/31

So why aren't comments being made to the PR?
